src/Symfony/Component/Validator/Constraints/Json.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
+ *
+ * @author Imad ZAIRIG <imadzairig@gmail.com>
+ */
+class Json extends Constraint
+{
+    const NOT_JSON_ERROR = '60d2f30b-8cfa-4372-b155-9656634de120';
```

Comment:
https://github.com/symfony/symfony/blob/3f1951af459f786ed94815e0c2a4121f17c1ff6c/src/Symfony/Component/Validator/Constraints/IsNull.php#L24

Error codes must be unique accros all constraints, so a new UUID should be used here.

https://www.uuidgenerator.net/